stable/redis/templates/svc-external-access.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.cluster.externalAccess.enabled }}
+{{- $fullName := include "redis.fullname" . }}
+{{- $nodesCount := .Values.cluster.nodes | int }}
+{{- $root := . }}
+
+{{- range $i, $e := until $nodesCount }}
+{{- $targetPod := printf "%s-%d" (printf "%s" $fullName) $i }}
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "redis.fullname" $ }}-{{ $i }}-svc
+  labels: {{- include "redis.labels" $ | nindent 4 }}
+    pod: {{ $targetPod }}
+  annotations: {{ include "redis.tplValue" ( dict "value" $root.Values.cluster.externalAccess.service.annotations "context" $) | nindent 4 }}
+spec:
+  type: {{ $root.Values.cluster.externalAccess.service.type }}
+  {{- if eq $root.Values.cluster.externalAccess.service.type "LoadBalancer" }}
```

Comment:
In principle, if this is enabled it's because it has to be LoadBalancer. Do you see another use case?